packages/server-wallet/src/chain-service/chain-service.ts

```diff
@@ -104,7 +104,7 @@ export class ChainService implements ChainServiceInterface {
 
     this.nitroAdjudicator.on(ChallengeRegistered, (...args) => {
       const event = getChallengeRegisteredEvent(args);
-      this.addFinalizingChannel(event.channelId, event.finalizesAt);
+      this.addFinalizingChannel({channelId: event.channelId, finalizesAtS: event.finalizesAt});
```

Comment:
Is `finalizesAtS` a typo or intentional? What is the S for?

Reply:
`S` stands for seconds as opposed to milliseconds. Javascript times are conventionally in milliseconds. Block times are in seconds. So I wanted to avoid comparing milliseconds timestamps with seconds timestamps. But perhaps `S` is not descriptive enough.